akka-actor/src/main/scala/akka/routing/Resizer.scala

```diff
@@ -71,6 +94,7 @@ case object DefaultResizer {
       backoffRate = resizerConfig.getDouble("backoff-rate"),
       messagesPerResize = resizerConfig.getInt("messages-per-resize"))
 
+  @deprecated("User Resizer.fromConfig instead", since = "2.4.1")
```

Comment:
I think we should remove the deprecation. It still does the right thing. Perhaps not that useful, but we can't deprecated things in minor releases (2.4.1).